solutions/fully-configurable/provider.tf

```diff
@@ -0,0 +1,31 @@
+provider "ibm" {
+  ibmcloud_api_key = var.ibmcloud_api_key
+}
+
+provider "kubernetes" {
+  host  = data.ibm_container_cluster_config.cluster_config.host
+  token = data.ibm_container_cluster_config.cluster_config.token
```

Comment:
your missing cert (needed for IKS clusters):
`cluster_ca_certificate = data.ibm_container_cluster_config.cluster_config.ca_certificate`